src/styles/layouts/_layout-authentication.scss

```diff
@@ -36,6 +41,13 @@
 
     @include respond-above('md') {
       padding: 4.5% 6%;
+
+      background: {
+        image: url('/images/sign-up-background.jpg');
+        repeat: no-repeat;
+        size: cover;
+        position: top 10% center;
```

Comment:
No need for `top`.
```suggestion
        position: center;
```

Reply:
I think it's more like the design when the top goes down. What do you think?

Reply:
It's fine for me, what do others think?